nlp_models/similizer_test.py

```diff
@@ -0,0 +1,32 @@
+import unittest
```

Comment:
Need more working tests, not comments, create only few simple tests for now and check with db